tests/tools/CMakeLists.txt

```diff
@@ -2,7 +2,6 @@
 # Licensed under the MIT License.
 
 if (OE_SGX)
-  add_subdirectory(oecert)
```

Comment:
oesign has both the tool itself, and tests for the tool (which is why the oesign directory is added in line 6).
Are there no tests for oecert?  Should a separate issue be filed for that or is testing of it already covered by some other test?  (It's not good to have things published to /opt/openenclave/bin that don't have tests.)

Reply:
There's a unit test `host_verify` that uses `oecert` as part of the test. There isn't a separate test for `oecert`. That's why `tests/tools/oecert` is removed and `tools/oecert` seems more appropriate.

Reply:
where does `host_verify` use `oecert`?  I can only find `_verify_cert()` in `host_verify\host\host.cpp` but I can't find any invocation of `oecert`

Reply:
Ah, I was wrong. It was part of the ACC Host Verification Test in Jenkins:
https://github.com/openenclave/openenclave/blob/8238dd84a3b10ded58fa3bb7ddd3ca17b81de048/.jenkins/pipelines/Azure/Linux/Jenkinsfile#L158